src/tests/ftest/dfuse/read.py

```diff
@@ -96,3 +96,98 @@ def test_dfuse_pre_read(self):
         )
 
         self.assertEqual(data["inodes"], 3, "expected 3 inodes in cache")
+
+
+class DFuseReadTest(DfuseTestBase):
+    """Base ReadTest test class.
+
+    :avocado: recursive
+    """
+
+    def test_dfuse_read(self):
+        """
+        Test Description:
+            Run a simple Write/Read test to check for read caching.
+
+        Write a file, then read from it and verify that there were no reads at the dfuse level.
+
+        Evict the file, read from it twice and verify the second read comes from cache.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=vm
+        :avocado: tags=dfuse,dfs
+        :avocado: tags=DFuseReadTest,test_dfuse_read
+        """
+
+        pool = self.get_pool(connect=False)
+        container = self.get_container(pool)
+
+        self.load_dfuse(self.hostlist_clients, None)
+
+        self.dfuse.disable_wb_cache.value = True
+
+        self.dfuse.env["D_LOG_MASK"] = "INFO,DFUSE=DEBUG"
+        self.dfuse.env["DD_MASK"] = "ALL"
+        self.dfuse.env["DD_SUBSYS"] = "ALL"
```

Comment:
Do you want this set for both test cases?
If so, we can put this in the config
```
dfuse:
  disable_wb_cache: true
  env_vars:
    - D_LOG_MASK=INFO,DFUSE=DEBUG
    - DD_MASK=ALL
    - DD_SUBSYS=ALL
```


And if it needs to be different, I'd recommend different namespaces like
```
dfuse:
  disable_wb_cache: true
dfuse_debug:
  disable_wb_cache: true
  env_vars:
    - D_LOG_MASK=INFO,DFUSE=DEBUG
    - DD_MASK=ALL
    - DD_SUBSYS=ALL
```

And then initialize like
```
self.load_dfuse(self.hostlist_clients, '/run/dfuse/*')
self.load_dfuse(self.hostlist_clients, '/run/dfuse_debug/*')
```